### PR TITLE
creates models

### PR DESCRIPTION
**Pull request by george-silva**

### CODE REVIEW

File: src/planscape/datasets/models.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+
+class DataLayerStatus(models.TextChoices):
+    READY = "READY", "Ready"
+    PENDING = "PENDING", "Pending"
+    FAILED = "FAILED", "Failed"
+
+
+class GeometryType(models.TextChoices):
+    NO_GEOM = "NO_GEOM", "No Geometry Field"
```
Comment (pflopez) [inline src/planscape/datasets/models.py:94]: when does this apply? can we have data layers without geometry?

File: src/planscape/datasets/models.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    RASTER = "RASTER", "Raster"
+    POINT = "POINT", "Point"
+    MULTIPOINT = "MULTIPOINT", "MultiPoint"
+    LINESTRING = "LINESTRING", "LineString"
+    MULTILINESTRING = "MULTILINESTRING", "MultiLineString"
+    POLYGON = "POLYGON", "MultiPolygon"
```
Comment (pflopez) [inline src/planscape/datasets/models.py:100]: Should this be `POLYGON = "POLYGON", "Polygon"` ?

Also- should we have this distintion (poly vs multi ) or is this something we should simplify and transform so we only deal with one type? 


Same with `linestring / MultiLineString` (i dont know what is the difference so not sure that makes sense)

**Review by roquebetioljr — APPROVED** [review]

File: src/planscape/datasets/models.py (hunk 1 above)
Comment (pflopez) [reply src/planscape/datasets/models.py:94]: I see that we are allowing `null` on this field on DataLayer so why do we need this value?

File: src/planscape/datasets/models.py (hunk 2 above)
Comment (pflopez) [reply src/planscape/datasets/models.py:100]: @george-silva still seeing POLYGON = "POLYGON", "MultiPolygon" here and on the migrations

**Review by pflopez — APPROVED** [review]

**Pull request merged**